### PR TITLE
Add Swedish language to HOWTO!

### PR DESCRIPTION
**Pull request by CxrlosKenobi**
## What does this PR do?
Add Swedish language to HOWTO

### Why is this valuable (or not)?
Swedish is spoken in more tha 5 countries, along the world, support the language embrace the reach for all people!

## Checklist:
- [x ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
@CxrlosKenobi are you Swedish native or is your mother language?

**Comment by CxrlosKenobi** [comment]
> @CxrlosKenobi are you Swedish native or is your mother language?

None of those, I spent some time with natives though and learned.

**Comment by eshellman** [comment]
Tack så mycket, and welcome to free-programming-books!

File: HOWTO-se.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Läs detta på andra språk: [English](HOWTO.md), [Français](HOWTO-fr.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Svenska](HOWTO-se.md).
+
+Välkommen till *Free-Programming-Books*! Vi välkomnar varmt nya medarbetare, även de som gör sin första Pull Request (PR) på GitHub. Om du är en av dem finns här några resurser som kan hjälpa dig:
+
+* [Om Pull begäran](https://docs.github.com/es/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+* [Skama en Pull begäran](https://docs.github.com/es/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
```
Comment (davorpa) [inline HOWTO-se.md:6]: why `es`

File: HOWTO-se.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* [Youtube - GitHub -handledning för nybörjare](https://www.youtube.com/watch?v=0fKg7e37bQE) *(på engelska)*
+* [Youtube - Hur man gafflar ett GitHub -arkiv och skickar en pull -begäran](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(på engelska)*
+* [Youtube - Curso intensivo de Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo) *(på engelska)*
+
+Var aldrig i tvivel, eller var rädd för att ställa frågor; varje bidragsgivare som du ser i förvaret började på sin tid med en första PR. Tänk om det är vår tusen-tusendel!
+
+Om du har erfarenhet som bidragsgivare på andra projekt med öppen källkod finns det några saker du kan göra för att få det att fungera. När den skickats till PR, utför * GitHub Actions * en * linter *; som hittar en meny för att hitta små problem med utrymme, utrymme, syntax eller läskunnighet. Om denna slutliga integrationsprocess ska slutföras kommer ljuset och allt är klart för din granskning; men om inte, klicka på "Detaljer för detaljer" som ger det exakta genomsnittet av det du tappade. Lösningen på detta problem och summan av förändringarna i din PR innebär ett nytt engagemang.
```
Comment (davorpa) [inline HOWTO-se.md:14]: Format are lost

**Review by davorpa — COMMENTED** [review]
It seems more like a translation made by Google translate due to losing markdown format

To reviewed at #5762

**Pull request merged**